database/seeds/WikisSeeder.php

```diff
@@ -1,44 +1,89 @@
 <?php
 
+use App\QueryserviceNamespace;
+use App\User;
 use App\Wiki;
+use App\WikiDb;
 use App\WikiDomain;
 use App\WikiManager;
 use App\WikiSetting;
+use App\WikiSiteStats;
 use Illuminate\Database\Seeder;
-use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Hash;
 use Illuminate\Support\Str;
 
 class WikisSeeder extends Seeder
 {
-    public function run()
+    /**
+     * @param \App\User $user
+     * @param string $name
+     * @param array $stats
+     */
+    public function createWiki($user, $name, $stats)
     {
-        // Just create one for now
-        // TODO this is all done in the WikiController but should probably be factored out so it can be reused...
+        $domain = $name . '.nodomain.example';
+
         $wiki = Wiki::create([
-            'sitename' => 'seededSite',
-            'domain' => 'seededsite.nodomain.dev',
+            'sitename' => $name,
+            'domain' => $domain
         ]);
+
         WikiDomain::create([
-            'domain' => 'seededsite.nodomain.dev',
-            'wiki_id' => $wiki->id,
+            'domain' => $domain,
+            'wiki_id' => $wiki->id
+        ]);
+
+        $index = substr(bin2hex(random_bytes(48)), 0, 10);
+
+        WikiDb::create([
+            'name' => 'mwdb_' . $index,
+            'user' => 'mwu_' . $index,
+            'password' => $index,
+            'version' => Config::get('wbstack.wiki_db_use_version'),
+            'prefix' => 'mwt_' . $index,
+            'wiki_id' => $wiki->id
+        ]);
+
+        QueryserviceNamespace::create([
```

Comment:
My gut feeling said that we only need to create the simplest of wikis (meaning name, user, version, id... etc)
I don't know if we really need other things? We only need it for testing and other stuff is just redundant. Unless you want to use the in other cases

Reply:
Frederik used to bashed me for adding things that we "will use in the future but not now" and the future sometimes come sooner than we expected.
So I will agree to keep them here. It doesn't hurt but I will keep it in mind